.github/workflows/quadrat_deploy.yml

```diff
@@ -20,15 +20,15 @@ jobs:
         uses: SamKirkland/FTP-Deploy-Action@3.1.1
         with:
           ftp-server: sftp://sftp.pressable.com:22/htdocs/wp-content/themes/
-          ftp-username: ${{ secrets.SB_DEMO_USERNAME }}
-          ftp-password: ${{ secrets.SB_DEMO_PASSWORD }}
+          ftp-username: ${{ secrets.QUADRAT_DEMO_USERNAME }}
+          ftp-password: ${{ secrets.QUADRAT_DEMO_PASSWORD }}
           git-ftp-args: --insecure
-          local-dir: ./seedlet-blocks/
+          local-dir: ./quadrat/
       - name: Deploy-Parent
         uses: SamKirkland/FTP-Deploy-Action@3.1.1
```

Comment:
I should have asked this question a long time ago, but does this action delete files that are no longer in the theme? Say if we rename or remove a template?

Reply:
I poked around the docs and couldn't tell for sure but... I THINK so.
The only way to know for sure is to remove something and see though.  In [v4 there is a "clean target" option](https://github.com/SamKirkland/FTP-Deploy-Action#settings) (which the docs imply shouldn't be necessary) but we're[ using v3 ](https://github.com/SamKirkland/FTP-Deploy-Action/tree/3.1.1)as v4 doesn't work with the host we're uploading to (Pressable).

Reply:
I see. Bless Sam Kirkland.